nextcloudappstore/core/models.py

```diff
@@ -56,18 +56,12 @@ def can_delete(self, user: User) -> bool:
 
     @staticmethod
     def search(lang, terms):
```

Comment:
TL;DR: do we get anything valuable in return that justifies the manager and does it play well with parler